Generic Types, Traits and Lifetime/Generic Types, Traits and Lifetime/Use Option/task-info.yaml

```diff
@@ -0,0 +1,21 @@
+type: edu
+files:
+- name: src/main.rs
+  visible: true
+  placeholders:
+  - offset: 130
+    length: 8
+    placeholder_text: 13
+  - offset: 158
+    length: 8
+    placeholder_text: 99
+  - offset: 207
+    length: 15
+    placeholder_text: // Something goes here
```

Comment:
An open placeholder with "//" affects all of the remaining line, leaving following stuff commented. I suggest using a "/**/" inline placeholder to avoid this